R/getPrevalence.R

```diff
@@ -511,14 +501,17 @@ setMethod("getPrevalentAbundance", signature = c(x = "SummarizedExperiment"),
 
 ############################# agglomerateByPrevalence ##########################
 
-#' @rdname agglomerate-methods
-#'   
-#' @param update.tree \code{TRUE} or \code{FALSE}: should
-#'   \code{rowTree()} also be agglomerated? (Default:
-#'   \code{update.tree = FALSE})
+#' Agglomerate data based on population prevalence
+#' 
+#' @rdname agglomerateByPrevalence
+#'  
+#' @inheritParams agglomerateByRank
 #' 
-#' @param other.label A single \code{character} valued used as the label for the
-#'   summary of non-prevalent taxa. (default: \code{other.label = "Other"})
+#' @param update.tree \code{Logical scalar}. Should
+#'   \code{rowTree()} also be agglomerated? (Default: \code{FALSE})
+#' 
+#' @param other.label \code{Character scalar}. Used as the label for the
+#'   summary of non-prevalent taxa. (default: \code{"Other"})
```

Comment:
addDominant has "other.name" which is essentially the same as other.label here. --> Can you replace other.label with other.name in this functions?